### ISSUE TITLE
__init__.py:10-12: Implement Manager access control by...

### ISSUE THREAD

**Issue by 0pdd** [timelesslounge/timelessis#176]
The puzzle `22-a5b13305` from #22 has to be resolved: 

https://github.com/timelesslounge/timelessis/blob/ae4c725b13e7afb9f349eb20f55e5a2d22cd4ebb/timeless/access_control/__init__.py#L10-L12

The puzzle was created by Vatavuk on 09-Feb-19. 

  

If you have any technical questions, don't ask me, submit new tickets instead. The task will be \"done\" when the problem is fixed and the text of the puzzle is _removed_ from the source code. Here is more about [PDD](http://www.yegor256.com/2009/03/04/pdd.html) and [about me](http://www.yegor256.com/2017/04/05/pdd-in-action.html).

**Comment by 0crat** [comment]
@emilianodellacasa[/z](https://www.0crat.com/u/emilianodellacasa) please, pay attention to this issue

**Comment by 0crat** [comment]
Job [#176](https://github.com/timelesslounge/timelessis/issues/176) is now in scope, role is `DEV`

**Comment by 0crat** [comment]
The job [#176](https://github.com/timelesslounge/timelessis/issues/176) assigned to @krzyk[/z](https://www.0crat.com/u/krzyk), here is [why](https://www.0crat.com/footprint/CF9LG1YCA/fa50ae88-9294-4b21-9626-75dabb3dfc8f); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html); there will be a monetary reward for this job

**Comment by 0crat** [comment]
@krzyk[/z](https://www.0crat.com/u/krzyk) this job was assigned to you 5days ago. It will be taken away from you soon, unless you close it, see [§8](http://www.zerocracy.com/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by krzyk** [comment]
@emilianodellacasa Regarding manager role:
```
Manager

    * Create / modify / activate / deactivate accounts for all managers, masters, interns associated with Company he/she belongs to;

    * View self-profile

    * View profiles of managers, masters, interns associated with Company he/she belongs to;
```

I assume Manager shouldn't be able to change/view other manager profiles right? So only masters and interns, right?

**Comment by krzyk** [comment]
@0crat waiting for review of PR #281

**Comment by 0crat** [comment]
> @0crat waiting for review of PR #281  ([here](https://github.com/timelesslounge/timelessis/issues/176#issuecomment-464518250))

@krzyk The impediment for [#176](https://github.com/timelesslounge/timelessis/issues/176) was registered successfully by @krzyk[/z](https://www.0crat.com/u/krzyk)

**Comment by 0pdd** [comment]
@0pdd the puzzle [#298](https://github.com/timelesslounge/timelessis/issues/298) is still not solved.

**Comment by 0pdd** [comment]
The puzzle `22-a5b13305` has disappeared from the source code, that's why I closed this issue.

**Comment by 0crat** [comment]
@ypshenychka[/z](https://www.0crat.com/u/ypshenychka) please review this job completed by @krzyk[/z](https://www.0crat.com/u/krzyk), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#176](https://github.com/timelesslounge/timelessis/issues/176) is now out of scope

**Comment by ypshenychka** [comment]
@0crat quality good

**Comment by 0crat** [comment]
Order was finished, quality is "good": +35 point(s) just awarded to @krzyk[/z](https://www.0crat.com/u/krzyk)

**Comment by 0crat** [comment]
Quality review completed: +8 point(s) just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)

**Comment by 0pdd** [comment]
@0pdd 2 puzzles [#298](https://github.com/timelesslounge/timelessis/issues/298), [#341](https://github.com/timelesslounge/timelessis/issues/341) are still not solved.

**Comment by 0pdd** [comment]
@0pdd the puzzle [#341](https://github.com/timelesslounge/timelessis/issues/341) is still not solved; solved: [#298](https://github.com/timelesslounge/timelessis/issues/298).

**Comment by 0pdd** [comment]
@0pdd the puzzle [#397](https://github.com/timelesslounge/timelessis/issues/397) is still not solved; solved: [#298](https://github.com/timelesslounge/timelessis/issues/298), [#341](https://github.com/timelesslounge/timelessis/issues/341).

**Comment by 0pdd** [comment]
@0pdd the puzzle [#481](https://github.com/timelesslounge/timelessis/issues/481) is still not solved; solved: [#298](https://github.com/timelesslounge/timelessis/issues/298), [#341](https://github.com/timelesslounge/timelessis/issues/341), [#397](https://github.com/timelesslounge/timelessis/issues/397).
